qutip/tests/test_gates.py

```diff
@@ -283,6 +285,35 @@ def _rand_3qubit_gate(N=None, controls=None, k=None):
                         o2 = psi_ref_out.overlap(psi_ref_in)
                         assert_(abs(o1 - o2) < 1e-12)
 
+    def test_expand_oper(self):
```

Comment:
Add docstring. This way the test comment will be printed to screen if verbose unit testing is chosen.